test/integration/eslint/test/index.test.js

```diff
@@ -429,5 +430,22 @@ describe('ESLint', () => {
       expect(stdout).toContain('<script src="https://example.com" />')
       expect(stdout).toContain('2 warnings found')
     })
+
+    test('file flag selectively lints files', async () => {
+      const { stdout, stderr } = await nextLint(
+        dirFileLinting,
+        ['--file', 'utils/bar.js'],
```

Comment:
Can you add a test for multiple files as well 👍

Reply:
Done 🚀